electron/src/utils.ts

```diff
@@ -34,7 +34,7 @@ export function isMac() {
 }
 
 export function isMacAppStore() {
-  return isMac() && process.mas && process.mas === true;
+  return isMac() && (process.mas ?? false);
```

Comment:
Not sure I understand this change, `process.mas` from electron's docs state's it's going to be `true` or `undefined`, therefore the `=== true` seems like the more correct test vs. using _truthy_.

Perhaps instead use:
```suggestion
  return isMac() && process.mas === true;
```

Reply:
Here's a better example of why it's returning `undefined`

![image](https://user-images.githubusercontent.com/3904944/159964473-d74daa89-cbbd-41eb-8808-f5b90ddc1c9f.png)